src/frontend/src/components/SmsDataTable/SmsDataTable.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
This module should make it easier to re-use the CBS datatable without the boilerplate

Reply:
this hack is to get around the (lack of) typescript support for Carbon React components

See here, https://github.com/carbon-design-system/carbon/discussions/10752#discussioncomment-4710978

Reply:
@Ryan-Cleminson @willyyto when/if you guys add any pages with data tables this should be handy